webrtc.html

```diff
@@ -4847,13 +4837,10 @@ <h2>Methods</h2>
               <p>We define the <dfn>CollectTransceivers</dfn> algorithm as
               follows:</p>
               <ol>
-                <li>Let <var>transceivers</var> be a new sequence that
-                represents a snapshot of all the
-                <code><a>RTCRtpTransceiver</a></code> objects in this <code><a>
-                  RTCPeerConnection</a></code> object's <a>set of
-                  transceivers</a>. The conversion from the transceiver set to
-                  the sequence is user agent defined and the order does not
-                  have to be stable between calls.
+                <li>Let <var>transceivers</var> be a new sequence consisting of
+                all <code><a>RTCRtpTransceiver</a></code> objects in this
+                <code><a>RTCPeerConnection</a></code> object's
+                <a>set of transceivers</a>, in insertion order.
```

Comment:
Is `insertion order` defined somewhere?

Reply:
It's the order our algorithms add transceivers to the *set of transceivers*. This seems well-defined in our spec for local "add*" methods, and in JSEP for ones added by remote media descriptions.

Reply:
works for me!